spec/jobs/resend_validation_emails_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe Jobs::ResendValidationEmails do
+  before do
+    Jobs.run_immediately!
+  end
+
+  it 'should email inactive users' do
+    user = Fabricate(:user, active: false, created_at: 3.days.ago)
+
+    expect { described_class.new.execute({}) }
+      .to change { EmailLog.count }.by(1)
```

Comment:
This is not testing that we've sent the email but rather that we've created a new `EmailLog` record.

Can we instead write a test that will properly test that we've sent an email?